test/cli-alert/src/index.ts

```
@@ -64,6 +72,24 @@ async function discoverConsecutiveFailures(
   return failedJobs;
 }
 
+async function sendPagerDuty() {
+  await event({
+    data: {
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      routing_key: PD_ROUTING_KEY,
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      event_action: 'trigger',
+      payload: {
+        summary: 'CLI Alert. Smoke tests failing',
+        source: 'Snyk CLI Smoke tests',
+        severity: 'warning',
+      },
+    },
+  })
+    .then(console.log)
+    .catch(console.error);
```

If you are `await`ing, then it's easier to reason about it with `try/catch` and 

```js
try {
  const pdResponse = await event() 
  console.log(pdResponse)
} catch (error) {
  console.error(error)
}
```

Or maybe not even catching it and just let it throw to kill the run?